docs/input-manifest/api-reference.md

```diff
@@ -248,6 +248,30 @@ Dynamic nodepools are defined for cloud provider machines that Claudie is expect
   
   Autoscaler configuration for this nodepool. Mutually exclusive with `count`.
 
+- `labels`
+
+  Map of user defined labels, which will be applied on every node in the node pool. This field is optional.
+
+  By default, Claudie applies following labels on every node in the cluster.
```

Comment:
should mention that the user defined labels are applied in addition to the default labels ? I can imagine that it might be confusing

Same with the defaults `taints`